nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/src/main/java/org/nuxeo/ecm/platform/rendering/template/DocumentModelAdapter.java

```diff
@@ -74,11 +76,24 @@ public TemplateModel get(String key) throws TemplateModelException {
         // may be a schema name (doc.dublincore.title)
         Map<String, Object> properties = doc.getProperties(key);
         if (properties != null) {
-            return wrapper.wrap(properties);
+            return wrapper.wrap(unPrefixedMap(properties));
         }
         return wrapper.wrap(null);
     }
 
+    private static Map<String, Object> unPrefixedMap(Map<String, Object> props) {
+        Map<String, Object> res = new HashMap<String, Object>();
+        for (Entry<String, Object> e : props.entrySet()) {
+            String key = e.getKey();
+            int pos = key.indexOf(':');
+            if (pos > -1) {
+                key = key.substring(pos + 1);
+            }
+            res.put(key, e.getValue());
+        }
+        return res;
+    }
+
```

Comment:
wdyt if we replace by stream 

` return props.entrySet()
                    .stream()
                    .collect(Collectors.toMap(
                            e -> StringUtils.substring(e.getKey(), e.getKey().indexOf(":") + 1, e.getKey().length()),
                            Function.identity()));
`
StringUtils.substring(e.getKey(), e.getKey().indexOf(":") + 1, e.getKey().length()) will replace the control on the presence or not of `:`

Reply:
I've restored code from 6.0, so I will not apply this suggestion